app/routes/dette-har-vi-gjort/$slug.tsx

```diff
@@ -0,0 +1,51 @@
+import { json, LoaderFunction, MetaFunction } from "@remix-run/node";
+import { useLoaderData } from "@remix-run/react";
+import sanityClient from "~/sanity/sanity-client";
+
+// TODO: I propose to move this function to a shared utility/common file
+export function assertItemFound<T>(item: T | undefined): asserts item is T {
+  if (item === undefined)
+    throw new Response("Not Found", {
+      status: 404,
+    });
+}
+
+const query = (slug: string) =>
+  sanityClient.getAll("selvskryt", `slug.current == "${slug}"`);
+
+type Data = { item: Awaited<ReturnType<typeof query>>[0] };
+export const loader: LoaderFunction = async ({ params }) => {
+  const item = (await query(params.slug ?? ""))[0];
+  assertItemFound(item);
+
+  return json<Data>({ item });
+};
+
+export const meta: MetaFunction = ({ data }: { data: Data }) => ({
+  title: data.item.helmetTitle,
+  description: data.item.helmetDescription,
+});
+
+export default function () {
+  const { item } = useLoaderData<Data>();
+
+  // Quick and dirty
+  // Extract some text from the BlockContent body
+  const texts = item.body
+    ?.map((x) => (x as any)?.children[0]?.text)
```

Comment:
🗒️ Det genererte skjemaet var litt manglende når det kom til `BlockContent` definisjonen, dette er et problem uansett hvordan sanity klient oppsett vi går for.

Reply:
Absolutt kan den mapping koden være bedre! 😂  Ser bra ut 👍 

> Det blir vel uansett litt pain ja

I gatsby siden så brukes det `PortableText` -> `@sanity/block-content-to-react` komponent for å rendre block content ser det ut som
https://github.com/capraconsulting/capraconsulting.no/blob/master/packages/web/src/components/portableText/portableText.tsx
https://github.com/sanity-io/block-content-to-react

Den er deprecated ser det ut som, og den nye er: https://github.com/portabletext/react-portabletext
Så hvis vi kan bare ploppe inn rikteksten vi ønsker å rendre inn ditt, så går vell greit at typene er litt manglefulle, sålenge sanity sitt komponent håndterer det

Reply:
Ah, det er nice! :clap: Da betyr ikke typingen så mye på vår side, nei :blush: